.github/CODEOWNERS

```diff
@@ -211,6 +211,8 @@ build-utils/        @getsentry/owners-js-build
 # To find matching files -> find . -name "*sentry_app*.py"
 *sentry_app*.py                               @getsentry/ecosystem
 *sentryapp*.py                                @getsentry/ecosystem
+*doc_integration*.py                          @getsentry/ecosystem
+*docintegration*.py                           @getsentry/ecosystem
```

Comment:
Why are there two variants of it?

I guess `sentry_app` is like that too 😬

Reply:
Looking at the relevant files, the `*docintegration*.py` rule really only applies to the DB model in `docintegrationavatar.py`. I tried to use `*doc_integration*.py` as the name for most other files (endpoints, tests, serializers).

Since I saw `sentry_app` does it, I figured it was acceptable but if we prefer I can explicitly add a rule for:
```
/src/sentry/models/docintegrationavatar.py     @getsentry/ecosystem
```
and leave
```
*doc_integration*.py                          @getsentry/ecosystem
```
as is

Reply:
If there is a convention doc or something I should be following when adding to CODEOWNERS, let me know!